packages/@coorpacademy-components/src/molecule/video-player/test/fixtures/jwplayer.js

```diff
@@ -14,6 +14,9 @@ export default {
         height: '100%'
       }
     },
+    scriptErrorMessage:
+      'Could not load the video player. Get sure to whitelist this platform within your adblocker if you use one.',
```

Comment:
Make sure

Reply:
`If you're using one`

Reply:
I've added the mooc wording: 'An error occurred while loading the video. Please make sure you have authorized this platform on your adblocker or deactivate your adblocker.',